packages/standalone-plugins/bundle/config/overlay/airgapped-network.yaml

```
@@ -0,0 +1,29 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+
+---
+apiVersion: cli.tanzu.vmware.com/v1alpha1
+kind: CLIPlugin
+metadata:
+  name: airgapped-network
+  annotations:
+    kapp.k14s.io/change-rule: "upsert after upserting cliplugins.cli.tanzu.vmware.com/crd"
+spec:
+  artifacts:
+    #@yaml/text-templated-strings
+    (@= data.values.airgapped-network.version @):
```

In the `packages/standalone-plugins/values.template.yaml` file the key is without hyphen.
```suggestion
    (@= data.values.airgappednetwork.version @):
```